libs/sdk-ui-kit/src/Dialog/HubspotConversionTouchPointDialogBase.tsx

```diff
@@ -154,6 +160,7 @@ export const HubspotConversionTouchPointDialogBase: React.FC<IHubspotConversionT
         formId: hubspotFormId,
         target: `#${hubspotFormTargetId}`,
         locale: intl.locale.split("-").shift() as HubSpotFormLocale,
+        submitButtonClass,
```

Comment:
In this comment https://github.com/gooddata/gooddata-ui-sdk/pull/1825#discussion_r732667033 I just intended to suggest that we assign directly a text for submitButtonClass here, something like below and then in function `onDialogSubmit` we just replace `'hs-button.primary.large'` with `'gd.hs-button.primary'`

`submitButtonClass: 'gd.hs-button.primary'`

I didn't mean that we complicate the codes a bit by adding a new prop to `IHubspotConversionTouchPointDialogBaseProps`
※ The newly added prop is, IMO, almost never used by client, meaning we will almost end up using the default class name `'hs-button.primary.large'` instead. (My concern was that `'hs-button.primary.large'` might be changed in the future if we have to upgrade the package).

Reply:
This is a public property of the Hubspot form. I don't think it will changed frequently. But I have updated this. Thanks!